charon/src/translate/translate_ctx.rs

```diff
@@ -452,6 +452,9 @@ impl<'tcx, 'ctx> TranslateCtx<'tcx, 'ctx> {
             attributes: self.translate_attributes_from_rid(def_id),
             inline: self.translate_inline_from_rid(def_id),
             public,
+            opaque: self
+                .translate_attributes_from_rid(def_id)
+                .contains(&(String::from("charon::opaque"))),
```

Comment:
As per the description of the original issue (https://github.com/AeneasVerif/charon/issues/206) we also want to check for "aeneas::opaque"